enginetest/spatial_index_tests.go

```diff
@@ -316,6 +316,21 @@ var SpatialIndexTests = []SpatialIndexPlanTest{
 			},
 		},
 	},
+	{
+		name: "filter point table with st_within",
+		setup: []string{
+			"create table point_tbl(p point not null srid 0, spatial index (p))",
+			"insert into point_tbl values (point(0,0)), (point(1,1)), (point(2,2))",
+		},
+		tests: []SpatialIndexPlanTestAssertion{
+			{
+				q: "select p from point_tbl where st_within(p, point(0,0))",
```

Comment:
Just confirming, this validates that point_tbl is indexed? If someone breaks that in the future this will error to notify?

Reply:
should we have a test for null point? null value input into filter? can the index be a primary? the is a keyless table, maybe a pk table also? query that returns data?

Reply:
I can add tests for primary key table and a null point.